Hestia/app/src/androidTest/java/com/rugged/application/hestia/BackendInteractorTest.java

```diff
@@ -11,38 +10,40 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-
 import java.util.ArrayList;
-
+import hestia.UI.DeviceListFragment;
 import hestia.backend.Activator;
 import hestia.backend.ActivatorState;
 import hestia.backend.BackendInteractor;
 import hestia.backend.Device;
-
+import hestia.backend.DevicesChangeListener;
 import static org.junit.Assert.*;
 
 @RunWith(AndroidJUnit4.class)
 public class BackendInteractorTest {
```

Comment:
Avoid complexity in your tests _as much as possible_. Have simple small tests that are properly named and explained.
Debugging your debugging tool is not something you wish to do.